### PR TITLE
feat: add the ability to post api inventory items manually

### PR DESCRIPTION
**Pull request by danielvladco**
Add the ability to post api inventory items manually

### CODE REVIEW

File: backend/pkg/rest/apiinventory_controller.go [hunk 1]
```diff
@@ -26,6 +26,17 @@ import (
 	_database "github.com/apiclarity/apiclarity/backend/pkg/database"
 )
 
+func (s *Server) PostAPIInventory(params operations.PostAPIInventoryParams) middleware.Responder {
+	apiInfo := &_database.APIInfo{
+		Type: params.Body.APIType,
+		Name: params.Body.Name,
+		Port: params.Body.Port,
+	}
+	s.dbHandler.APIInventoryTable().CreateAPIInfo(apiInfo)
```
Comment (FrimIdan) [inline backend/pkg/rest/apiinventory_controller.go:35]: maybe we should not ignore the error if it's data that we are getting from the user.

Comment (FrimIdan) [reply backend/pkg/rest/apiinventory_controller.go:35]: What if that the api already exist, isn't it better to use `FirstOrCreate`? Or to verify that it does not exits (name+port+type) before creating a new one?

Comment (danielvladco) [reply backend/pkg/rest/apiinventory_controller.go:35]: I like the idea with FirstOrCreate more, the apis should be idempotent

Comment (danielvladco) [reply backend/pkg/rest/apiinventory_controller.go]: done

**Review by FrimIdan — APPROVED** [review]

**Review by FrimIdan — APPROVED** [review]

File: api/swagger.yaml [hunk 2]
```diff
@@ -452,6 +463,21 @@ paths:
                   $ref: '#/definitions/ApiInfo'
         default:
           $ref: '#/responses/UnknownError'
+    post:
+      summary: 'Create API inventory item'
+      parameters:
+        - in: body
+          name: body
+          schema:
+            $ref: '#/definitions/CreateApiInfo'
```
Comment (fishkerez) [inline api/swagger.yaml:472]: Why not using ApiInfo?

Comment (danielvladco) [reply api/swagger.yaml:472]: because we need to provide apiType

Comment (fishkerez) [reply api/swagger.yaml:472]: then I think it will be better if CreateApiInfo will contain ApiInfo and type instead of duplicating ApiInfo fields.

Comment (fishkerez) [reply api/swagger.yaml:472]: And I will probably would not call it CreateApiInfo, maybe something more general like ApiInfoWithType that can be reused later?

**Review by fishkerez — APPROVED** [review]

**Review by akpsgit — APPROVED** [review]

**Pull request merged**